module/bulkUploadEntities/helper.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * name : bulkUploadEntities/helper.js
```

Comment:
@rakeshSgr - Create different helper files inside the entities module, no need of a new module here.